### ISSUE TITLE
Jekyll does not build after install pagemaster

### ISSUE THREAD

**Issue by dshowland** [mnyrop/pagemaster#24]
I installed pagemaster per the instructions in the README and now I cannot build or serve jekyll.
After running `bundle install` and then `bundle exec jekyll serve` I get this error:

```
LoadError: cannot load such file -- /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/pagemaster-2.1.0/lib/pagemaster/collection
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/pagemaster-2.1.0/lib/pagemaster.rb:14:in `require_relative'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/pagemaster-2.1.0/lib/pagemaster.rb:14:in `<top (required)>'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:74:in `require'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:74:in `block (2 levels) in require'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `each'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `block in require'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `each'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `require'
  /Users/me/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:174:in `require'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/jekyll-4.0.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/jekyll-4.0.0/exe/jekyll:11:in `<top (required)>'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `load'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `<top (required)>'
```

Here's my gemfile:
```
# frozen_string_literal: true

source "https://rubygems.org"

git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

# gem "rails"

gem "jekyll", "~> 4.0"

group :jekyll_plugins do
  gem 'pagemaster'
end
```

I've followed a few troubleshooting steps related to this error and they all come back to something wrong with pagemaster.

**Comment by mnyrop** [comment]
Did you add 
```
plugins:
  - pagemaster
```
to your `_config.yml` file? If not, can you add it and try rerunning `bundle exec jekyll pagemaster` ?

**Comment by dshowland** [comment]
Yes, here's `config.yml`:

```
plugins:
  - pagemaster

collections:
  collection-name:
    output: true
    source: data-file
    id_key: token
    layout: default```

**Comment by mnyrop** [comment]
Apologies. It looks like there was an issue relatively requiring symlinks in plugins that wasn't caught in the tests. I removed `2.1.0` and pushed a (hopefully fixed) `2.1.1`

**Comment by dshowland** [comment]
No worries, thanks for the quick attention to the issue!

I have updated to `2.1.1` but I'm still getting the same error message when I run `bundle exec jekyll build` or `bundle exec jekyll pagemaster`

```
bundler: failed to load command: jekyll (/Users/me/Development/project/vendor/bundle/ruby/2.6.0/bin/jekyll)
LoadError: cannot load such file -- pagemaster/collection
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/pagemaster-2.1.1/lib/pagemaster.rb:14:in `require'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/pagemaster-2.1.1/lib/pagemaster.rb:14:in `<top (required)>'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:74:in `require'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:74:in `block (2 levels) in require'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `each'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `block in require'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `each'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `require'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:174:in `require'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/jekyll-4.0.0/lib/jekyll/plugin_manager.rb:53:in `require_from_bundler'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/gems/jekyll-4.0.0/exe/jekyll:11:in `<top (required)>'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `load'
  /Users/me/Development/project/vendor/bundle/ruby/2.6.0/bin/jekyll:23:in `<top (required)>'
```
Gemfile.lock
```
GEM
  remote: https://rubygems.org/
  specs:
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    colorator (1.1.0)
    concurrent-ruby (1.1.5)
    em-websocket (0.5.1)
      eventmachine (>= 0.12.9)
      http_parser.rb (~> 0.6.0)
    eventmachine (1.2.7)
    ffi (1.12.1)
    forwardable-extended (2.6.0)
    http_parser.rb (0.6.0)
    i18n (1.8.2)
      concurrent-ruby (~> 1.0)
    jekyll (4.0.0)
      addressable (~> 2.4)
      colorator (~> 1.0)
      em-websocket (~> 0.5)
      i18n (>= 0.9.5, < 2)
      jekyll-sass-converter (~> 2.0)
      jekyll-watch (~> 2.0)
      kramdown (~> 2.1)
      kramdown-parser-gfm (~> 1.0)
      liquid (~> 4.0)
      mercenary (~> 0.3.3)
      pathutil (~> 0.9)
      rouge (~> 3.0)
      safe_yaml (~> 1.0)
      terminal-table (~> 1.8)
    jekyll-sass-converter (2.0.1)
      sassc (> 2.0.1, < 3.0)
    jekyll-watch (2.2.1)
      listen (~> 3.0)
    kramdown (2.1.0)
    kramdown-parser-gfm (1.1.0)
      kramdown (~> 2.0)
    liquid (4.0.3)
    listen (3.2.1)
      rb-fsevent (~> 0.10, >= 0.10.3)
      rb-inotify (~> 0.9, >= 0.9.10)
    mercenary (0.3.6)
    pagemaster (2.1.1)
      jekyll (~> 4)
      rainbow (~> 3.0)
      safe_yaml (~> 1.0)
    pathutil (0.16.2)
      forwardable-extended (~> 2.6)
    public_suffix (4.0.3)
    rainbow (3.0.0)
    rb-fsevent (0.10.3)
    rb-inotify (0.10.1)
      ffi (~> 1.0)
    rouge (3.15.0)
    safe_yaml (1.0.5)
    sassc (2.2.1)
      ffi (~> 1.9)
    terminal-table (1.8.0)
      unicode-display_width (~> 1.1, >= 1.1.1)
    unicode-display_width (1.6.0)

PLATFORMS
  ruby

DEPENDENCIES
  jekyll (~> 4.0)
  pagemaster

BUNDLED WITH
   2.1.4
```

**Comment by mnyrop** [comment]
This is so strange! I'm not able to reproduce the error when using the gem locally. While I work on a fix, can you see if it works in the meantime when you replace your Gemfile with the git branch instead of the published gem, like so:

```rb
group :jekyll_plugins do
  gem 'pagemaster', git: 'https://github.com/mnyrop/pagemaster', branch: '2.1.1'
end
```

?

**Comment by dshowland** [comment]
Replacing the published gem with the git branch worked!

**Comment by mnyrop** [comment]
Hi @dshowland, this has, i believe, *actually* been fixed as of 2.1.2. I just tested it with the published gem, so using just `gem 'pagemater'` in your Gemfile and running `bundle update` should do the trick. Thanks for reporting the bug.
